source/apiVolontaria/volunteer/serializers.py

```diff
@@ -37,3 +43,157 @@ class Meta:
         read_only_fields = [
             'id',
         ]
+
+
+class CellBasicSerializer(serializers.ModelSerializer):
+
+    """
+
+    This class represents the Cell model serializer.
+
+    A serializer will format the object prior to the interaction with the user.
```

Comment:
I think we don't need this comment here, It's the same things for all serializers in this projects.
Maybe we can explain it in the documentation for developers in the ReadTheDoc website.

Reply:
I agree